mordecai3/geoparse.py

```diff
@@ -325,12 +325,13 @@ def pick_event_loc(self, d):
         return d
 
 
-    def geoparse_doc(self, 
-                     text, 
-                     plover_cat=None, 
-                     debug=False, 
-                     trim=True, 
-                     known_country=None,
+    def geoparse_doc(self,
+                     text,
+                     plover_cat=None,
+                     debug=False,
+                     trim=True,
+                     include_countries:list[str] | None=None,
+                     exclude_countries: list[str] | None = None,
```

Comment:
and here

If there is a type hint, there should be spaces surrounding `=` of default value. If no type hint, then no spaces.

Reply:
Done